src/bentoml/_internal/configuration/default_configuration.yaml

```
@@ -38,6 +38,8 @@ api_server:
     metrics:
       host: 0.0.0.0
       port: 3001
+  health_probe:
+    check_runners: True
```

very nit pick:
```yaml
```suggestion
  probe:
    health:
      check_runners: true
```
```

---

Seoyeon and I discussed and went a different route. Let us know if you have any questions.